<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BillsManagerClass</class>
 <widget class="QWidget" name="BillsManagerClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>596</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>BillsManager</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QListWidget" name="listWidget"/>
     </item>
     <item>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QPushButton" name="btnSort">
         <property name="text">
          <string>Sort</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QFormLayout" name="formLayout">
       <item row="3" column="0">
        <widget class="QLabel" name="labelPaid">
         <property name="text">
          <string>Paid:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="labelSum">
         <property name="text">
          <string>Sum:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="labelSerial">
         <property name="text">
          <string>Serial no.:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="labelCompany">
         <property name="text">
          <string>Company:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="lineEditCompany"/>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="lineEditSerial"/>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="lineEditSum"/>
       </item>
       <item row="3" column="1">
        <widget class="QLineEdit" name="lineEditPaid"/>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="btnTotal">
       <property name="text">
        <string>Compute total</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelTotalSum">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="billsmanager.qrc"/>
 </resources>
 <connections/>
</ui>
